test/extended/networking/services.go

```diff
@@ -65,4 +71,159 @@ var _ = Describe("[sig-network] services", func() {
 			Expect(checkServiceConnectivity(f1, f2, DIFFERENT_NODE)).To(Succeed())
 		})
 	})
+
+	var retryInterval = 1 * time.Minute
+
+	Context("external ip", func() {
+		It("ensures policy is configured correctly on the cluster [Serial]", func() {
+			var wg1, wg2 sync.WaitGroup
+			namespace := oc.Namespace()
+			adminConfigClient := oc.AdminConfigClient()
+			k8sClient := oc.KubeClient()
+			serviceClient := k8sClient.CoreV1().Services(namespace)
+			// Test a load balancer service with default cluster networks config
+			// In this case service creation must throw an error for non admin user
+			By("create service of type load balancer with default cluster networks config")
+			serviceName := "svc-without-ext-ip"
+			labelMap1 := make(map[string]string)
+			labelMap1["foo1"] = "bar1"
+			createPodForService(k8sClient, namespace, serviceName, "", labelMap1)
```

Comment:
I don't think you need to create a pod here, since we're just testing service creation.